libs/screen/image.cpp

```diff
@@ -1206,7 +1206,7 @@ void nextYRange_HighUp(int x, LineGenState *line, ValueRange *yRange) {
         --line->y;
     }
 }
-
+// simular with sub-function LineHigh() of DrawLine(), but will yield back after each all Y values of given X calculating complete. Next time called will continue going with the state when it yield back.
```

Comment:
```suggestion
// This function is similar to the sub-function drawLineHigh for drawLine. However, it yields back after calculating all Y values of a given X. When the function is called again, it continues from the state where it yielded back previously.
```

Let me know if this takes away from the original meaning.

I didn't realize this was mimicking a previous pattern (drawLine calling drawLineHigh). @riknoll should we follow a naming convention here?